commands/navigation/bear.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.author Jax0rz
+# @authorURL https://github.com/Jax0rz
+# @description Replace your owe token:https://bear.app/faq/X-callback-url%20Scheme%20documentation/#token-generation
+# @raycast.schemaVersion 1
+# @raycast.title Bear
+# @raycast.mode silent
+# @raycast.packageName Navigation
+
+# Optional parameters:
+# @raycast.icon images/bear.png
+# @raycast.argument1 { "type": "text", "placeholder": "query","percentEncoded": true}
+
+open "bear://x-callback-url/search?term=${1// /%20}&show_window=yes&token=979709-0C37A6-F84D4D"
```

Comment:
@Jax0rz we have one last thing to do here

**What to do:**
```suggestion
open "bear://x-callback-url/search?term=$1&show_window=yes&token=979709-0C37A6-F84D4D"
```


As the user will need to it's own token, you will have to follow the instructions defined in the guideline. Check it out here: https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-additional-modification